### ISSUE TITLE
Unable to export table partitions from mysql using ora2pg

### ISSUE THREAD

**Issue by sanyamsinghal** [darold/ora2pg#1399]
It detects 4 partitions but could not export DDLs for them and continues

I am using the export_schema.sh script which comes with the standard ora2pg project.

```
[root@ip-10-9-80-202 sakila]# ./export_schema.sh
[========================>] 33/33 tables (100.0%) end of scanning.
[========================>] 11/11 objects types (100.0%) end of objects auditing.
Running: ora2pg -p -m -t TABLE -o table.sql -b ./schema/tables -c ./config/ora2pg.conf
[========================>] 33/33 tables (100.0%) end of scanning.
[========================>] 40/40 tables (100.0%) end of table export.
Running: ora2pg -p -m -t VIEW -o view.sql -b ./schema/views -c ./config/ora2pg.conf
[========================>] 7/7 views (100.0%) end of output.
Running: ora2pg -p -m -t GRANT -o grant.sql -b ./schema/grants -c ./config/ora2pg.conf
WARNING: Exporting privilege as non DBA user is not allowed, see USER_GRANT
Running: ora2pg -p -m -t TRIGGER -o trigger.sql -b ./schema/triggers -c ./config/ora2pg.conf
[========================>] 6/6 triggers (100.0%) end of output.
Running: ora2pg -p -m -t FUNCTION -o function.sql -b ./schema/functions -c ./config/ora2pg.conf
[========================>] 3/3 functions (100.0%) end of functions export.
Running: ora2pg -p -m -t PROCEDURE -o procedure.sql -b ./schema/procedures -c ./config/ora2pg.conf
[========================>] 4/4 procedures (100.0%) end of procedures export.
Running: ora2pg -p -m -t PARTITION -o partition.sql -b ./schema/partitions -c ./config/ora2pg.conf
[>                        ] 0/4 partitions (0.0%) end of output.
Running: ora2pg -p -m -t DBLINK -o dblink.sql -b ./schema/dblinks -c ./config/ora2pg.conf
[========================>] 0/0 dblink (100.0%) end of output.
Running: ora2pg -t VIEW -o view.sql -b ./sources/views -c ./config/ora2pg.conf
[========================>] 7/7 views (100.0%) end of output.
Running: ora2pg -t TRIGGER -o trigger.sql -b ./sources/triggers -c ./config/ora2pg.conf
[========================>] 6/6 triggers (100.0%) end of output.
Running: ora2pg -t FUNCTION -o function.sql -b ./sources/functions -c ./config/ora2pg.conf
[========================>] 3/3 functions (100.0%) end of functions export.
Running: ora2pg -t PROCEDURE -o procedure.sql -b ./sources/procedures -c ./config/ora2pg.conf
[========================>] 4/4 procedures (100.0%) end of procedures export.
Running: ora2pg -t PARTITION -o partition.sql -b ./sources/partitions -c ./config/ora2pg.conf
[>                        ] 0/4 partitions (0.0%) end of output.
Running: ora2pg -t MVIEW -o mview.sql -b ./sources/mviews -c ./config/ora2pg.conf
[========================>] 0/0 materialized views (100.0%) end of output.


To extract data use the following command:

ora2pg -t COPY -o data.sql -b ./data -c ./config/ora2pg.conf
```

@darold please have a look and let me know if something is wrong with my setup

**Comment by darold** [comment]
Can you post the output of the following command:
```
ora2pg -p -m -t PARTITION -o partition.sql -b ./schema/partitions -c ./config/ora2pg.conf -v
```
Also what is your setting of PG_VERSION in ora2pg.conf?

**Comment by sanyamsinghal** [comment]
@darold output is this:
```
$ ora2pg -p -m -t PARTITION -o partition.sql -b ./schema/partitions -c ./config/ora2pg.conf -v
Ora2Pg v23.1
```

PG_VERSION is **12** in my config file

**Comment by sanyamsinghal** [comment]
@darold any update on this?

**Comment by sanyamsinghal** [comment]
@darold is it only me who is having issues with Table Partitions in MySQL?

**Comment by gilles-migops** [comment]
It might be broken for everyone, this is just that I have not found the time to work on it until now.

**Comment by sanyamsinghal** [comment]
ok thanks, just wanted to confirm that.

**Comment by gilles-migops** [comment]
Please give a try to latest development code, commit 7a01b2d might solve this issue. I have no MySQL database with partitioning so I have just done a simple test with RANGE partitioning so let me know if there is any other issues.

**Comment by sanyamsinghal** [comment]
thanks a lot @darold, i will let you know once i test this.

**Comment by sanyamsinghal** [comment]
I am not sure if this has come to notice since this is a closed issue, please acknowledge. 
Will wait for acknowledgment for sometime before creating a new issue.

**Comment by darold** [comment]
Actually the work on exporting partitions for MySQL was never ended, it don't works at all. I will fix that.

**Comment by darold** [comment]
Commit eb5eb2b might fix MySQL partition export. I still have to work on sub partition but if you don't have such object you can test latest development code.

**Comment by sanyamsinghal** [comment]
Yeah, I don't have a subpartition as of now in my database. I will test this latest commit and let you know.

**Comment by sanyamsinghal** [comment]
Thanks, @darold it looks good for my use case(that has some tables with RANGE PARTITIONING)
Will like to try sub-partitioning also once those changes land.
